src/components/Markdown/index.js

```diff
@@ -0,0 +1,7 @@
+import ReactMarkdown from 'react-markdown'
+
+import React from 'react'
+
+export default ({children}) => (
```

Comment:
@ijones16 Good question. When I use a library I generally like to wrap it so that I can provide extra options or defaults if needed; also so I could potentially change the underlying implementation without breaking all of the uses (like I could switch to a different markdown library and nothing else would change other than the wrapper component). In my experience, wrapping libraries is usually a good idea.